### PR TITLE
Update to TheGraph v10 (Awaiting graph-node Bug Fix)

### PR DESCRIPTION
**Pull request by dOrgJelli**
Why update? Because v10 adds support for data source templates that are needed to track newly deployed contracts.

More info about the feature here: https://github.com/graphprotocol/graph-node/pull/832

A conversation around this update has been taking place here: https://github.com/daostack/subgraph/issues/197

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,11 +43,12 @@
 		"eslint-plugin-standard": "^4.0.0",
 		"ipfs-http-client": "^29.1.1",
 		"spawn-command": "0.0.2-1",
-		"watch": "^1.0.2"
+		"watch": "^1.0.2",
+		"ws": "^6.2.1"
```
Comment (orenyodfat) [inline package.json]: does this is a must ?

**Comment by orenyodfat** [comment]
lets wait till this [issue/bug](https://github.com/graphprotocol/graph-node/issues/896) will be cleared.
as we are using this type of queries to monitor the graph.

File: package.json (hunk 1 above)
Comment (dOrgJelli) [reply package.json]: The CI build failed without: https://github.com/daostack/subgraph/runs/112147150

Comment (orenyodfat) [reply package.json]: `ws` is a dependency of `subscriptions-transport-ws` and `web3`  and it is pulled automatically . there is no need to add that to package.json.
see : 
Screen Shot 2019-04-28 at 0 05 45

please update package-lock.json  : `rm package-lock.json` and `npm i`

Comment (dOrgJelli) [reply package.json]: That did the trick, thank you!

**Comment by orenyodfat** [comment]
note : all these unrelevants commits will need to be squashed on the event of merging it.

**Comment by dOrgJelli** [comment]
@orenyodfat the issue posted above has been closed, should this be merged?

**Comment by orenyodfat** [comment]
> @orenyodfat the issue posted above has been closed, should this be merged?

well, there is already v 11. which I am currently working on.
it is not straight fwd.

**Comment by dOrgJelli** [comment]
> > @orenyodfat the issue posted above has been closed, should this be merged?
> 
> well, there is already v 11. which I am currently working on.
> it is not straight fwd.
> see #220

Sounds good, feel free to close this. Also just a reminder that this PR is still up which adds `tempaltes:` support to the `generate-subgraph.js` script: https://github.com/daostack/subgraph/pull/215

**Comment by orenyodfat** [comment]
#220 update to graph-node v11

**Pull request closed without merging**